cluster/gce/config-common.sh

```diff
@@ -98,6 +98,14 @@ function get-cluster-ip-range {
   echo "${suggested_range}"
 }
 
+function get-alias-range-size() {
```

Comment:
Document this function, including  parameters

Reply:
Done.